app/components/portfolios/details_component.html.erb

```diff
@@ -25,7 +25,39 @@
           )
         end
 
-        # TODO: WP-68877 render progress bar
+        if render_sub_status_bar?
+          header.with_column(classes: "op-portfolios--progress") do
+            render(
+              Primer::Beta::ProgressBar.new(
+                size: :large,
+                data: {
+                  hover_card_trigger_target: "trigger",
+                  hover_card_popover_id: sub_status_hover_card_id,
+                  test_selector: "op-portfolios--sub-status-bar"
+                }
+              )
+            ) do |bar|
+              sub_statuses_with_percentages.each do |status|
+                status => { code:, percentage: }
+
+                status_label = t("js.grid.widgets.project_status.#{code || 'not_set'}")
+
+                bar.with_item(
+                  percentage:,
+                  bg: nil, # unset default color so that we can inject our own via CSS class
```

Comment:
Don't know what is preferred - unsetting default background or setting our own background with `!important`